### PR TITLE
#2817 upcoming design reviews

### LINKED ISSUE #2817: [Home Page] - Upcoming Design Reviews Section
### Description

We want a section where heads can see all the upcoming design reviews. This component will be a new page block on the heads view title ‘Upcoming Design Reviews’ followed by the number of items in the list. We define an upcoming design review as a design review that is scheduled within two weeks of the current date and is not done.
	
We currently do not have a design review card. Create a new component called DesignReviewCard.tsx that we will use to display upcoming design reviews. This card should contain the wbs name, date and time, location, and status of the design review.

### Acceptance Criteria

- [ ] new component is added to the head/lead home page that displays all upcoming design reviews for the head’s team
- [ ] Design review card is created

### Proposed Solution

You can access a team’s design reviews by accessing the projects in a team, which wbsElements that store design reviews
	
For the design review card, take a look at the WorkPackageCard and use that as a reference for creating a new card. Refer to the design review model in the prisma schema to take a look at the fields you can access on a design review to populate the card.

### Mocks

image

### PR DESCRIPTION
**Pull request by jessegarcia1**
## Changes

Created the card and display for the upcoming design reviews for leads.

## Screenshots

image
image
image

Closes #2817

### CODE REVIEW

**Review by walker-sean — CHANGES_REQUESTED** [review]
Alright it looks great but two things I noticed:
1. When there are too many design reviews everything gets too smushed. Talk to @caiodasilva2005 about using the component with the scrollbar
2. You should also include the person that scheduled the design review, as they may not think to add themself as a required member.

**Review by caiodasilva2005 — CHANGES_REQUESTED** [review]
If a design review is scheduled for Thursday 12/12 for example, it is showing Wednesday 12/11 on the design review card in the component, meaning every card is showing the design review as one day before it is scheduled. Try potentially modifying the get Weekday and get removeYear functions to account for having to add 1 to the day, and make sure to test edge cases like scheduling a review on Sunday

File: src/frontend/src/pages/HomePage/components/DesignReviewCard.tsx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+      )}
+    </Stack>
+  );
+};
+
+const DisplayStatus: React.FC<DesignReviewProps> = ({ designReview, user }) => {
+  const history = useHistory();
+
+  const confirmedMemberIds = designReview.confirmedMembers.map((user) => user.userId);
+  console.log('CONFIRMED:', confirmedMemberIds);
```
Comment (walker-sean) [inline src/frontend/src/pages/HomePage/components/DesignReviewCard.tsx]: Remove

File: src/frontend/src/pages/HomePage/components/DesignReviewCard.tsx [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+          Confirm Availibility
+        </NERButton>
+      ) : (
+        <Typography mr={1}>{designReview.status}</Typography>
+      )}
+    </>
+  );
+};
+
+function getWeekday(date: Date): string {
```
Comment (walker-sean) [inline src/frontend/src/pages/HomePage/components/DesignReviewCard.tsx]: Could you change these to const functions

**Review by walker-sean — CHANGES_REQUESTED** [review]

**Review by walker-sean — APPROVED** [review]
LGTM

**Pull request merged**